HelpSource/Guides/WritingUGens.schelp

```diff
@@ -107,8 +106,7 @@ architecture as well as a bunch of helper functions. These are documented at lin
 
 subsection:: The Entry Point
 
-When the library is loaded the server calls a function in the library, which is defined by the code::PluginLoad()::
-macro.  This entry point has two responsibilities:
+When the library is loaded the server calls a function in the library, which is defined by the code::PluginLoad():: macro.  This entry point has two responsibilities:
```

Comment:
Also I don't believe we should unwrap the lines - but the double space before "This" could be removed.